### ISSUE TITLE
When I click on a report from a contact profile, I see a flash of the History tab before getting to the report view

### ISSUE THREAD

**Issue by sglangevin** [medic/medic-webapp#2834]
To reproduce, go to any contact profile that has something in the History. Click on one of the reports in the History section - you'll see a flash of the History tab, followed by viewing the report you had clicked on. The correct behavior is to navigate directly to the report without seeing the History tab.

Definitely happens on mobile - might be harder to notice on web.

**Comment by SCdF** [comment]
On my Nexus 5 it only happens the first time (apologies I don't have touch view turned on so it's a bit hard to follow): https://dl2.pushbulletusercontent.com/pqSerd1m4uDbnBnxuanZRDEEQGCdY2hQ/demo2.mp4

On my worst tecno it's a little more noticeable, but honestly not much more so.

This is almost certainly a side effect of our current hierarchy, which is also why when you hit back from filling it you go to the reports tab instead of back to the contact or contacts tab or whatever. I think both of these issues are covered in #2763 and #2600 respectively.

**Comment by garethbowen** [comment]
PR: https://github.com/medic/medic-webapp/pull/2843

The problem was when a user kicks off a new search we immediately show the LHS so they can see the spinner while the search is loading. Unfortunately we also do that when the search is kicked off on first load of the controller. I've added some code to detect when the search is being execute explicitly as opposed to in the background.

**Comment by SCdF** [comment]
Works for me!
